bundle/compliance/cis_azure/rules/cis_7_2/test.rego

```diff
@@ -0,0 +1,121 @@
+package compliance.cis_azure.rules.cis_7_2
+
+import data.compliance.policy.azure.data_adapter
+import data.lib.test
+
+valid_managed_disk = {
+	"id": "/subscriptions/sub-id/resourceGroups/cloudbeat-resource-group-1695893762/providers/Microsoft.Compute/disks/cloudbeatVM_OsDisk_1_e736df07f12142a9a2784ea8de9084ce",
+	"resourceGroup": "cloudbeat-resource-group-1695893762",
+	"storageAccountType": "Standard_LRS",
+}
+
+generate_vm(managed_disk) = {
```

Comment:
should be under test_data

Reply:
Should I really move it if it's only used in this file?